server/src/test/java/org/elasticsearch/cluster/routing/allocation/FailedShardsRoutingTests.java

```diff
@@ -507,10 +506,14 @@ public void testRebalanceFailure() {
         RoutingNodes routingNodes = clusterState.getRoutingNodes();
 
         assertThat(clusterState.routingTable().index("test").size(), equalTo(2));
-        assertThat(routingNodes.node("node1").numberOfShardsWithState(STARTED, RELOCATING), equalTo(2));
-        assertThat(routingNodes.node("node1").numberOfShardsWithState(STARTED), lessThan(3));
```

Comment:
This and following `lessThan(3)` assertions are removed as they are covered by assertion above